tasks/request_from_archive/request_from_archive.py

```diff
@@ -3,6 +3,7 @@
 Description:  Lambda function that makes a restore request from archive bucket for each input file.
 """
 import json
+import logging
```

Comment:
Since that is technology-specific, it breaks clean architecture.

Reply:
hmm I see. Looks like we are breaking clean architecture for now in our lambdas since all lambdas now use powertools

Reply:
Using the logger is fine. This `import` is only relevant for the type-hint on the new function, making the function technology-agnostic since it just knows it accepts a `logging.logger`, but doesn't need to know about the Powertools logger.